CHANGELOG.md

```diff
@@ -80,4 +80,8 @@ This will avoid infinitely looping over pages until an empty page is discovered.
 * UGCShareConnection.retrieveUGCPostsByAuthors() now includes sortBy param to get posts
 sorted by creation date.
 
-## 1.0.8 (Work in progress)
\ No newline at end of file
+## 1.0.8 (Work in progress)
```

Comment:
```suggestion
## 1.0.8
```
Once this is merged into DEV it will trigger a release, please add the next version as work in progress.

Reply:
Thank you I have update the pom version.